cropper/app/controllers/Application.scala

```diff
@@ -34,6 +34,9 @@ object Application extends Controller with ArgoHelpers {
 
   val mediaApiKey = keyStore.findKey("cropper").getOrElse(throw new Error("Missing cropper API key in key bucket"))
 
+  val parseExportRequest = parse.json(ExportRequest.readExportRequest)
```

Comment:
I really don't like those forms: We have [something similar here](https://github.com/guardian/grid/blob/master/metadata-editor/app/controllers/EditsController.scala#L174-L179).

I would like to explore typed requests e.g.
```
Authenticated.async(parse.json)[Export] { export =>
  //...
}
```